### PR TITLE
Server and db update - Maker Fixes

### PR DESCRIPTION
**Pull request by VeraTs**
Added Maker endpoint, changed enum names

### CODE REVIEW

File: CheckerServer/CheckerServer/Models/DishType.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿namespace CheckerServer.Models
 {
-    public enum DishType
+    public enum eDishType
```
Comment (AvivBk) [inline CheckerServer/CheckerServer/Models/DishType.cs:3]: used unknown instead, but ill change it

File: CheckerServer/CheckerServer/Models/Order.cs [hunk 2]
```diff
@@ -7,24 +7,24 @@ public class Order : BaseDBItem
...
-        public OrderStatus Status { get; set; } = OrderStatus.Ordered;
+        public eOrderStatus Status { get; set; } = eOrderStatus.Ordered;
         public List<OrderItem> Items { get; set; } = new List<OrderItem>();
-        public OrderType OrderType { get; set; } = OrderType.FIFO;
+        public eOrderType OrderType { get; set; } = eOrderType.FIFO;
         public float TotalCost { get; set; }
         public float RemainsToPay { get; set; }
     }
 
-    public enum OrderStatus
```
Comment (AvivBk) [inline CheckerServer/CheckerServer/Models/Order.cs:17]: i don't know if that's matter but all my enums are on a separate directory

File: CheckerServer/CheckerServer/Models/OrderItem.cs [hunk 3]
```diff
@@ -15,11 +15,11 @@ public class OrderItem : BaseDBItem
...
-        public LineItemStatus LineStatus { get; set; } = LineItemStatus.Locked;
+        public eItemStatus Status { get; set; } = eItemStatus.Ordered;
+        public eLineItemStatus LineStatus { get; set; } = eLineItemStatus.Locked;
     }
 
-    public enum ItemStatus      // the status of the order item overall
+    public enum eItemStatus      // the status of the order item overall
     {
         Ordered,                // initial state
         AtLine,                 // during entire time it is at preperation line
```
Comment (AvivBk) [inline CheckerServer/CheckerServer/Models/OrderItem.cs:25]: In-Progress instead At-Line

File: CheckerServer/CheckerServer/Models/OrderItem.cs [hunk 4]
```diff
@@ -28,7 +28,7 @@ public enum ItemStatus      // the status of the order item overall
         Returned                // customer returned item for some reason
     }
 
-    public enum LineItemStatus  // status of the order item in the line of preperation 
+    public enum eLineItemStatus  // status of the order item in the line of preperation 
     {
         Locked,
```
Comment (AvivBk) [inline CheckerServer/CheckerServer/Models/OrderItem.cs:33]: Available instead of To-Do

File: CheckerServer/CheckerServer/Models/Order.cs (hunk 2 above)
Comment (VeraTs) [reply CheckerServer/CheckerServer/Models/Order.cs:17]: It doesn't matter, in this case.

File: CheckerServer/CheckerServer/Models/OrderItem.cs (hunk 3 above)
Comment (VeraTs) [reply CheckerServer/CheckerServer/Models/OrderItem.cs:25]: AtLine is what was approved previously for clarity, I see no need to change now.

File: CheckerServer/CheckerServer/Models/OrderItem.cs (hunk 4 above)
Comment (VeraTs) [reply CheckerServer/CheckerServer/Models/OrderItem.cs:33]: As this is meant to reflect the list it belongs to in the Line, ( 1 of 3 lists), Keeping the name the same as the list name is meaningful.

**Pull request closed without merging**